frontend/src/features/auth/icons/SignUpBackground.tsx

```
@@ -0,0 +1,85 @@
+const SignUpBackground = () => {
+  return (
+    <svg
+      className='overflow-visible'
+      width='900'
+      height='900'
+      viewBox='0 0 900 900'
+      fill='none'
+      xmlns='http://www.w3.org/2000/svg'
+    >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)